.babelrc

```diff
@@ -54,7 +54,18 @@
         }], "stage-3", "flow"],
       "plugins": [
         "transform-object-rest-spread",
-        "transform-class-properties"
+        "transform-class-properties",
+        [
+          "module-resolver",
+          {
+            "root": [
+              "./"
+            ],
+            "alias": {
+              "components": "./src/webui/src/components"
```

Comment:
This place is not correct if you intend resolve ` "components": "./src/webui/src/components"` it should be done in the `ui` environment.